orchestrators/ProcurementOperations/tests/Unit/Coordinators/GoodsReceiptCoordinatorTest.php

```
@@ -0,0 +1,96 @@
+            receivedBy: 'user-1',
+            lineItems: [[
+                'poLineId' => 'line-1',
+                'productId' => 'prod-1',
+                'quantityReceived' => 5.0,
+                'uom' => 'EA',
+            ]],
+        );
+
+        $coordinator = new GoodsReceiptCoordinator(
+            contextProvider: new class () {
+                public function getPreReceiptContext(string $tenantId, string $purchaseOrderId, string $warehouseId, array $lineItems): never
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$warehouseId'. (undefined)

(UnusedFormalParameter)